### ISSUE TITLE
Operator console version v0.14.2 leaves t.Spec.Image empty for a freshly deployed tenant

### ISSUE THREAD

**Issue by harshavardhana** [minio/operator#1941]
After creating a tenant through the UI 

```
~ kubectl minio tenant list

Tenant 'store', Namespace 'store', Total capacity 4.0 GiB

  Current status: Initialized 
  MinIO version:  
```
The `MinIO version:` is empty and this is not empty with `kubectl minio tenant create --help` 

This is a regression from previous release AFAICS.

**Comment by bexsoft** [comment]
Transferred to a proper repository

**Comment by allanrogerr** [comment]
This is resolved in the latest version 5.0.13. PTAL and try again. See https://github.com/allanrogerr/public/wiki/operator%E2%80%901941 for a test guide.
